.github/workflows/deploy.yml

```
@@ -23,10 +23,16 @@ jobs:
         uses: actions/checkout@v3
         with:
           persist-credentials: false
+      - uses: awalsh128/cache-apt-pkgs-action@v1
+        with:
+          packages: calibre
+          version: 1.0
       - name: Install and Build
         run: |
           npm install
           npx honkit build
+          npx honkit epub
+          mv book_*.epub _book/
```

I like the filename suggestion, so something like this?
```suggestion
          rename book django-girls-tutorial book_*.epub
          mv *.epub _book/
```

---

I can't seem to get that to work. I've tried:
- just this suggested change das-g@3d8a98dfe
  → `rename: command not found` ([failed workflow](https://github.com/das-g/django-girls-tutorial/actions/runs/4838171061))
- with Ubuntu package `rename` das-g@ec674e3
  → `Bareword "book" not allowed while "strict subs" in use at line 1, in:
book` ([failed workflow](https://github.com/das-g/django-girls-tutorial/actions/runs/4838214073/jobs/8622550060#step:4:248))
- with Ubuntu package `util-linux` & command `rename.ul` das-g@a988e99
  → `rename.ul: command not found` ([failed workflow](https://github.com/das-g/django-girls-tutorial/actions/runs/4838303349/jobs/8622679182#step:4:248))
- with Ubuntu package `util-linux` & command `rename` das-g@1fe6e5e
  → `rename: command not found` ([failed workflow](https://github.com/das-g/django-girls-tutorial/actions/runs/4838364548))
- with Ubuntu package `rename`, command `file-rename` and escaped quotes around the word `book` das-g@61ae7d1
  → ["successful" workflow](https://github.com/das-g/django-girls-tutorial/actions/runs/4838467289), but files not renamed, as can be seen [in the subsequent workflow](https://github.com/das-g/django-girls-tutorial/actions/runs/4838516940/jobs/8623011212#step:4:248) where I [made `mv` verbose](https://github.com/das-g/django-girls-tutorial/commit/c3246d7a83ec5ae5c6b534e43ed0f20e5b1d966a#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R36).

---

> I think my suggestion to call both rename and mv was a copy paste mistake.

Was it? We need the results to end up in the `_book/` directory for them to be deployed.

---

Ah, I wasn't aware that perl rename and util-linux rename have that much different conventions in how you specify the replacement.